Reqnroll/Bindings/IBindingInvoker.cs

```diff
@@ -4,10 +4,9 @@
 
 namespace Reqnroll.Bindings
 {
-    [Obsolete("Use async version of the interface (IAsyncBindingInvoker) whenever you can")]
+    [Obsolete("The synchronous code invocation API has been deprecated. Please use IAsyncBindingInvoker instead.", true)]
```

Comment:
proposal: remove, as it's also not used now (missing in the DI)

obsolete was introduced before Reqnroll 1.0 in https://github.com/reqnroll/Reqnroll/commit/8b7eb2e9c35c80b7c5f2c525f6b44019d421b669

Reply:
Yeah. We can remove it.

Reply:
removed